src/overlays/actors/ovl_Mir_Ray/z_mir_ray.c

```diff
@@ -247,11 +247,11 @@ void MirRay_SetupCollider(MirRay* this) {
     f32 y = (this->poolPt.y - this->sourcePt.y) * dataEntry->unk_10;
     f32 z = (this->poolPt.z - this->sourcePt.z) * dataEntry->unk_10;
 
-    this->collider1.elements[0].dim.worldSphere.center.x = this->sourcePt.x + x;
-    this->collider1.elements[0].dim.worldSphere.center.y = this->sourcePt.y + y;
-    this->collider1.elements[0].dim.worldSphere.center.z = this->sourcePt.z + z;
+    this->collider1.elements->dim.worldSphere.center.x = this->sourcePt.x + x;
+    this->collider1.elements->dim.worldSphere.center.y = this->sourcePt.y + y;
+    this->collider1.elements->dim.worldSphere.center.z = this->sourcePt.z + z;
 
-    this->collider1.elements[0].dim.worldSphere.radius = dataEntry->unk_14 * this->collider1.elements->dim.scale;
+    this->collider1.elements->dim.worldSphere.radius = dataEntry->unk_14 * this->collider1.elements->dim.scale;
```

Comment:
Wasn't the `[0]` right before?

Reply:
Yeah, but the weird thing is that the scale has to be accessed with `elements->` to match (can see that before the change). So personally I would prefer them all to have the `elements->`.

Reply:
fwiw this matches
```c
    {
        ColliderJntSphElement* jntSphElem = &this->collider1.elements[0];

        jntSphElem->dim.worldSphere.radius = dataEntry->unk_14 * jntSphElem->dim.scale;
    }
```
but idk

Reply:
Yeah I still prefer just the `elements->`. Still easier to see its the same access as the ones above.

Reply:
I don't have a strong preference either way. Paths are notorious for needing `->` instead of [0] so it's probably something we have to live with